Fix failing TestRemoteChartInstall

Noticed that TestRemoteChartInstall is constantly failing in CICD:

```
=== CONT  TestRemoteChartInstall
    install_test.go:78: 
        	Error Trace:	install_test.go:78
        	Error:      	Received unexpected error:
        	            	error while running command: exit status 1; WARNING: Kubernetes configuration file is group-readable. This is insecure. Location: /home/circleci/.kube/config
        	            	WARNING: Kubernetes configuration file is world-readable. This is insecure. Location: /home/circleci/.kube/config
        	            	Error: INSTALLATION FAILED: failed to download "terratest-0jn8ma/nginx" at version "9.7.4"
        	Test:       	TestRemoteChartInstall
TestRemoteChartInstall 2023-01-31T21:31:31Z logger.go:66: Running command helm with args [delete --namespace testremotechartinstall-vy4hh7 --timeout 5m1s nginx-osz4bd]
TestRemoteChartInstall 2023-01-31T21:31:31Z logger.go:66: WARNING: Kubernetes configuration file is group-readable. This is insecure. Location: /home/circleci/.kube/config
TestRemoteChartInstall 2023-01-31T21:31:31Z logger.go:66: WARNING: Kubernetes configuration file is world-readable. This is insecure. Location: /home/circleci/.kube/config
TestRemoteChartInstall 2023-01-31T21:31:32Z logger.go:66: Error: uninstall: Release not loaded: nginx-osz4bd: release: not found
    delete.go:11: 
        	Error Trace:	delete.go:11
        	            				panic.go:482
        	            				testing.go:864
        	            				install_test.go:78
        	Error:      	Received unexpected error:
        	            	error while running command: exit status 1; WARNING: Kubernetes configuration file is group-readable. This is insecure. Location: /home/circleci/.kube/config
        	            	WARNING: Kubernetes configuration file is world-readable. This is insecure. Location: /home/circleci/.kube/config
        	            	Error: uninstall: Release not loaded: nginx-osz4bd: release: not found
        	Test:       	TestRemoteChartInstall
```

--- a/modules/gcp/compute_test.go
+++ b/modules/gcp/compute_test.go
@@ -23,7 +23,7 @@ const DEFAULT_IMAGE_FAMILY_PROJECT_NAME = "ubuntu-os-cloud"
 const DEFAULT_IMAGE_FAMILY_NAME = "family/ubuntu-1804-lts"
 
 // Regions that don't support running f1-micro instances
-var RegionsToAvoid = []string{"asia-east2", "southamerica-west1", "europe-west8", "europe-southwest1"}
+var RegionsToAvoid = []string{"asia-east2", "southamerica-west1", "europe-west8", "europe-southwest1", "us-east5-a"}
 
 func TestGetPublicIpOfInstance(t *testing.T) {
 	t.Parallel()

--- a/modules/gcp/compute_test.go
+++ b/modules/gcp/compute_test.go
@@ -22,15 +22,15 @@ const DEFAULT_MACHINE_TYPE = "f1-micro"
 const DEFAULT_IMAGE_FAMILY_PROJECT_NAME = "ubuntu-os-cloud"
 const DEFAULT_IMAGE_FAMILY_NAME = "family/ubuntu-1804-lts"
 
-// Regions that don't support running f1-micro instances
-var RegionsToAvoid = []string{"asia-east2", "southamerica-west1", "europe-west8", "europe-southwest1", "us-east5-a"}
+// Zones that support running f1-micro instances
+var ZonesThatSupportF1Micro = []string{"us-central1-a", "us-east1-b", "us-west1-a", "europe-north1-a", "europe-west1-b", "europe-central2-a"}
 
 func TestGetPublicIpOfInstance(t *testing.T) {
 	t.Parallel()
 
 	instanceName := RandomValidGcpName()
 	projectID := GetGoogleProjectIDFromEnvVar(t)
-	zone := GetRandomZone(t, projectID, nil, nil, RegionsToAvoid)
+	zone := GetRandomZone(t, projectID, ZonesThatSupportF1Micro, nil, nil)
 
 	createComputeInstance(t, projectID, zone, instanceName)
 	defer deleteComputeInstance(t, projectID, zone, instanceName)
@@ -77,7 +77,7 @@ func TestGetAndSetLabels(t *testing.T) {
 	instanceName := RandomValidGcpName()
 	projectID := GetGoogleProjectIDFromEnvVar(t)
 
-	zone := GetRandomZone(t, projectID, nil, nil, RegionsToAvoid)
+	zone := GetRandomZone(t, projectID, ZonesThatSupportF1Micro, nil, nil)
 
 	createComputeInstance(t, projectID, zone, instanceName)
 	defer deleteComputeInstance(t, projectID, zone, instanceName)
@@ -113,7 +113,7 @@ func TestGetAndSetMetadata(t *testing.T) {
 	projectID := GetGoogleProjectIDFromEnvVar(t)
 	instanceName := RandomValidGcpName()
 
-	zone := GetRandomZone(t, projectID, nil, nil, RegionsToAvoid)
+	zone := GetRandomZone(t, projectID, ZonesThatSupportF1Micro, nil, nil)
 
 	// Create a new Compute Instance
 	createComputeInstance(t, projectID, zone, instanceName)

--- a/modules/helm/install_test.go
+++ b/modules/helm/install_test.go
@@ -26,7 +26,7 @@ import (
 const (
 	remoteChartSource  = "https://charts.bitnami.com/bitnami"
 	remoteChartName    = "nginx"
-	remoteChartVersion = "9.7.4"
+	remoteChartVersion = "13.2.23"
 )
 
 // Test that we can install a remote chart (e.g bitnami/nginx)

--- a/test/gcp/packer_gcp_basic_example_test.go
+++ b/test/gcp/packer_gcp_basic_example_test.go
@@ -22,6 +22,9 @@ var DefaultRetryablePackerErrors = map[string]string{
 }
 var DefaultTimeBetweenPackerRetries = 15 * time.Second
 
+// Regions that support running f1-micro instances
+var RegionsThatSupportF1Micro = []string{"us-central1", "us-east1", "us-west1", "europe-west1"}
+
 // Zones that support running f1-micro instances
 var ZonesThatSupportF1Micro = []string{"us-central1-a", "us-east1-b", "us-west1-a", "europe-north1-a", "europe-west1-b", "europe-central2-a"}
 

--- a/test/gcp/packer_gcp_basic_example_test.go
+++ b/test/gcp/packer_gcp_basic_example_test.go
@@ -22,8 +22,8 @@ var DefaultRetryablePackerErrors = map[string]string{
 }
 var DefaultTimeBetweenPackerRetries = 15 * time.Second
 
-// Regions that don't support n1-standard-1 instances
-var RegionsToAvoid = []string{"asia-east2", "southamerica-west1", "europe-west8", "europe-southwest1", "us-east5-a"}
+// Zones that support running f1-micro instances
+var ZonesThatSupportF1Micro = []string{"us-central1-a", "us-east1-b", "us-west1-a", "europe-north1-a", "europe-west1-b", "europe-central2-a"}
 
 const DefaultMaxPackerRetries = 3
 
@@ -35,7 +35,7 @@ func TestPackerGCPBasicExample(t *testing.T) {
 	projectID := gcp.GetGoogleProjectIDFromEnvVar(t)
 
 	// Pick a random GCP zone to test in. This helps ensure your code works in all regions.
-	zone := gcp.GetRandomZone(t, projectID, nil, nil, RegionsToAvoid)
+	zone := gcp.GetRandomZone(t, projectID, ZonesThatSupportF1Micro, nil, nil)
 
 	packerOptions := &packer.Options{
 		// The path to where the Packer template is located

--- a/test/gcp/packer_gcp_basic_example_test.go
+++ b/test/gcp/packer_gcp_basic_example_test.go
@@ -23,7 +23,7 @@ var DefaultRetryablePackerErrors = map[string]string{
 var DefaultTimeBetweenPackerRetries = 15 * time.Second
 
 // Regions that don't support n1-standard-1 instances
-var RegionsToAvoid = []string{"asia-east2", "southamerica-west1", "europe-west8", "europe-southwest1"}
+var RegionsToAvoid = []string{"asia-east2", "southamerica-west1", "europe-west8", "europe-southwest1", "us-east5-a"}
 
 const DefaultMaxPackerRetries = 3
 

--- a/test/gcp/terraform_gcp_example_test.go
+++ b/test/gcp/terraform_gcp_example_test.go
@@ -109,7 +109,7 @@ func TestSshAccessToComputeInstance(t *testing.T) {
 	// Setup values for our Terraform apply
 	projectID := gcp.GetGoogleProjectIDFromEnvVar(t)
 	randomValidGcpName := gcp.RandomValidGcpName()
-	zone := gcp.GetRandomZone(t, projectID, nil, nil, RegionsToAvoid)
+	zone := gcp.GetRandomZone(t, projectID, ZonesThatSupportF1Micro, nil, nil)
 
 	terraformOptions := &terraform.Options{
 		// The path to where our Terraform code is located

--- a/test/gcp/terraform_gcp_ig_example_test.go
+++ b/test/gcp/terraform_gcp_ig_example_test.go
@@ -24,7 +24,7 @@ func TestTerraformGcpInstanceGroupExample(t *testing.T) {
 	// Setup values for our Terraform apply
 	projectId := gcp.GetGoogleProjectIDFromEnvVar(t)
 
-	region := gcp.GetRandomRegion(t, projectId, nil, RegionsToAvoid)
+	region := gcp.GetRandomRegion(t, projectId, RegionsThatSupportF1Micro, nil)
 
 	randomValidGcpName := gcp.RandomValidGcpName()
 	clusterSize := 3

